src/Servers/HttpSys/src/HttpSysRequestTimingType.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Server.HttpSys;
+
+public enum HttpSysRequestTimingType
```

Comment:
Still need to document these. I've reached out to the http.sys team to see if they have anything already I can borrow from :)